### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by rizar**

### CODE REVIEW

File: pytorch_rl/LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Ilya Kostrikov
+Copyright (c) 2017 Université de Montréal
```
Comment (dmitriy-serdyuk) [inline pytorch_rl/LICENSE]: maybe MILA? not sure how it woks

**Comment by rizar** [comment]
@nouiz what is the right thing to write in LICENSE?

**Comment by maximecb** [comment]
Can't do that, this is a fork of:
https://github.com/ikostrikov/pytorch-a2c-ppo-acktr

If we end up writing our own RL code, we can have our own license.

**Comment by maximecb** [comment]
OTOH, we could modify the copyright in here, if that makes sense:
https://github.com/maximecb/baby-ai-game/blob/master/LICENSE

**Pull request closed without merging**